install_files/securedrop-app-code/debian/control

```diff
@@ -6,5 +6,5 @@ Homepage: https://securedrop.org
 
 Package: securedrop-app-code
 Architecture: amd64
-Depends: python-pip,apparmor-utils,gnupg2,haveged,python,secure-delete,sqlite3,${dist:Depends},libapache2-mod-wsgi,libapache2-mod-xsendfile,redis-server,supervisor,securedrop-keyring,securedrop-config,libpython2.7-dev
+Depends: ${dist:Depends}, apparmor-utils, gnupg2, haveged, libapache2-mod-xsendfile, libapache2-mod-wsgi (= 4.3.0-1.1build1), python3, python3-pip, redis-server, secure-delete, securedrop-config, securedrop-keyring, sqlite3, supervisor
```

Comment:
so if we deployed this change using #4355 then we wouldn't need to install `python3-pip` (problematic as it's from the universe channel) right? we wouldn't need to pip install the wheels because we'd ship a virtualenv cc @kushaldas @rmol @conorsch

Reply:
I've only read the [dh-virtualenv docs](https://dh-virtualenv.readthedocs.io/en/latest/index.html) but yes, it looks like that would be a better way to get this done.

Reply:
Once we switch to `dh-virtualenv`, will this change still require any other package dependency updates that conflict with the current cron-apt update logic (#3376)?